docs/topics/db/models.txt

```diff
@@ -1053,6 +1053,21 @@ For example::
         serves_hot_dogs = models.BooleanField(default=False)
         serves_pizza = models.BooleanField(default=False)
 
+Would be equivalent to::
+
+    from django.db import models
+
+    class Place(models.Model):
+        name = models.CharField(max_length=50)
+        address = models.CharField(max_length=80)
+
+    class Restaurant(Place):
```

Comment:
Ah, then in hindsight I could actually see this as being confused as a copy-paste example to use vs. for illustration purposes only.

I definitely think showing the mechanics of MTI is valuable though...